app/src/main/java/com/example/pinterbest/utilities/Validator.kt

```diff
@@ -0,0 +1,179 @@
+package com.example.pinterbest.utilities
+
+import android.util.Patterns
+import android.widget.EditText
+import com.google.android.material.textfield.TextInputLayout
+import java.util.regex.Pattern
+
+class Validator private constructor() {
+    companion object {
```

Comment:
you can use `object Validator {` to make the same behavior

Reply:
Fixed here: https://github.com/enginire4ek/Pinterbest/pull/1/commits/c349b66d96a4b6bbf03557b4bc3771de1638810b